### ISSUE TITLE
jasmine_node_test can pick up wrong jasmine-core version

### ISSUE THREAD

**Issue by alexeagle** [bazelbuild/rules_nodejs#632]
Repro:

clone https://github.com/alexeagle/platform/tree/brokenz
bazel test //modules/effects/spec:test

All specs pass, but then jasmine exits with code 3 so Bazel thinks it failed.

I added some console logging, and here's the stack at the point that our jasmine_runner.js gets the completion event from jasmine:

```
Error: should be passed
    at jrunner.onComplete (node_modules/@bazel/jasmine/src/jasmine_runner.js:97:15)
    at module.exports.jasmineDone (node_modules/@bazel/jasmine/node_modules/jasmine/lib/reporters/completion_reporter.js:11:5)
    at dispatch (node_modules/jasmine-core/lib/jasmine-core/jasmine.js:2042:28)
    at ReportDispatcher.jasmineDone (node_modules/jasmine-core/lib/jasmine-core/jasmine.js:2013:11)
    at node_modules/jasmine-core/lib/jasmine-core/jasmine.js:780:18
    at ZoneDelegate.invokeTask (node_modules/zone.js/dist/zone-node.js:421:31)
    at Zone.runTask (node_modules/zone.js/dist/zone-node.js:188:47)
    at drainMicroTaskQueue (node_modules/zone.js/dist/zone-node.js:595:35)
    at ZoneTask.invokeTask (node_modules/zone.js/dist/zone-node.js:500:21)
    at Timeout.ZoneTask.invoke (node_modules/zone.js/dist/zone-node.js:485:48)
    at Timeout.timer [as _onTimeout] (node_modules/zone.js/dist/zone-node.js:2040:29)
    at ontimeout (timers.js:424:11)
    at tryOnTimeout (timers.js:288:5)
    at listOnTimeout (timers.js:251:5)
    at Timer.processTimers (timers.js:211:10)
```

Note that we get the jasmine library from `node_modules/@bazel/jasmine/node_modules/jasmine` so this will be the right version that `@bazel/jasmine` depends on - but we are picking up `jasmine-core` from `node_modules/jasmine-core` so I think the version skew is causing this.

**Comment by gregmagolan** [comment]
Fixed by #633

**Comment by gregmagolan** [comment]
Test for this case added in #637
